### ISSUE TITLE
4.1.2 error on system install: Failed to initialize the site instance "kutna_hora".

### ISSUE THREAD

**Issue by akien-mga** [Andrettin/Wyrmsun#184]
After packaging 4.1.2 system-wide and running it, I get these errors when launching wyrmsun and the game stops:
```
[2021.01.26 12:47:17] Position (222, 141) of map template "earth_45_60_latitude_north" already has a site ("jihlava").
[2021.01.26 12:47:17] Failed to initialize the site instance "kutna_hora".
[2021.01.26 12:47:17] Error initializing the instances of the site class.
[2021.01.26 12:47:17] Error initializing database.
```

**Comment by Andrettin** [comment]
Possibly the same as this issue:
https://github.com/Andrettin/Wyrmgus/issues/160

**Comment by akien-mga** [comment]
Thanks, confirmed fixed in 4.1.3.
